src/main/java/com/openshift/jenkins/plugins/pipeline/TimedOpenShiftBasePostAction.java

```diff
@@ -12,8 +12,8 @@
 
     public TimedOpenShiftBasePostAction(String apiURL, String namespace, String authToken, String verbose, String waitTime, String waitUnit) {
         super(apiURL, namespace, authToken, verbose);
-        this.waitTime = waitTime;
-        this.waitUnit = waitUnit;
+        this.waitTime = waitTime.trim();
+        this.waitUnit = waitUnit.trim();
```

Comment:
won't jobs from older version pass in null for the waitUnit?  I seem to recall that during our reviews there.
If my recollection holds , you can get a NullPointerException.

other existing params may that have been added over time may fall into that category as well.

Hence, if the above holds, I think you should do a global replace for each field, which will look like this example using `waitUnit`:

```
waitUnit != null ? waitUnit.trim() : null;
```